documentation/inventory_file_v2.md

```diff
@@ -1,10 +1,14 @@
 # Inventory file version 2
 
-Version 2 of the inventory file is experimental and might experience breaking changes in future releases.
-
 ## Migrating to version 2
 
-Version 2 of the inventory file changes some terms and syntax. To convert to version 2, you must make these changes.
+Version 2 of the inventory file changes some terms and syntax. To convert to version 2, you must make these changes. This can be done manually by changing fields in the inventory file or automatically with a Bolt command.
```

Comment:
```suggestion
Version 2 of the inventory file changes some terms and syntax. To convert to version 2, you must make these changes. This can be done manually by changing fields in the inventory file or by running `bolt project migrate` at the root of your [Boltdir](bolt_project_directories.md#).
```